docs/customization.md

```diff
@@ -1,8 +1,8 @@
 # How do I exclude recommendations?
 It is not uncommon for a recommendation to need to be excluded from configurations or slightly adjusted. These resources have built-in support for these situations.
-Every resource contains an 'ExcludeList' property that allows you to exclude settings based on their recommendation ID. These IDs can be found in the CIS documentation available from [CIS](src/../cis_documentation.md) or the [documentation page](https://github.com/techservicesillinois/SecOps-Powershell-CISDSC/wiki/Benchmark-documentation) of this repository. Using these PDFs you should be able to identify your setting in question and locate its ID number. Enabling exclusions will look similar to the below example.
+Every resource contains an 'ExcludeList' property that allows you to exclude settings based on their recommendation ID. These IDs can be found in the CIS documentation available from [CIS](src/../cis_documentation.md). Using these documents you should be able to identify your setting in question and locate its ID number. Enabling exclusions will look similar to the below example.
```

Comment:
This link is broken (or what it's supposed to point to doesn't yet exist).

Reply:
fixed in latest push